wye/templates/base.html

```diff
@@ -61,7 +61,9 @@
                 {% block sub_header %}{% endblock %}
 
                 {% block breadcrumbs %}
-                    <a href="{% url 'home-page' %}">Home</a>
+                    <ol class="breadcrumb">
```

Comment:
@vnbang2003  This looks very ugly. I think we should decide other place for this.
![home pythonexpress](https://cloud.githubusercontent.com/assets/3719122/10762051/4a78c65a-7cea-11e5-869f-27d263a9dd4d.png)